packages/editor/src/store/test/selectors.js

```diff
@@ -1609,7 +1646,7 @@ describe( 'selectors', () => {
 				currentPost: {},
 			};
 
-			expect( isEditedPostEmpty( state ) ).toBe( false );
+			expect( isEditedPostEmpty( state ) ).toBe( true );
```

Comment:
Why did this change? Why are tests changed without explanation?

Reply:
Initial edits should be considered a dirtying operation.

See https://github.com/WordPress/gutenberg/pull/16932#discussion_r315556523